torch/csrc/jit/script/tree_views.h

```diff
@@ -558,6 +477,105 @@ struct Const : public Expr {
   }
 };
 
+////////////////////////////////////////////////////////////////////////////////
+// Helper nodes (mostly for function arguments)
```

Comment:
Had to move this to use `Const` from inside `Param` to add `Param::has_default()` and `Param::default_value()`